src/utils/form.ts

```diff
@@ -44,7 +44,7 @@ export const mastodon = fb.group({
       },
       async (ctrl) => {
         const userBalance = await getBalance(ctrl.value);
-        if (userBalance.free < 1) {
+        if (userBalance.free < 0.001) {
```

Comment:
don't think we need this change here as the qrcode won't appear for the users to fund their accounts in that case

Reply:
updated